### ISSUE TITLE
CMake: perl swig MakeMaker script depends on GNUMake

### ISSUE THREAD

**Issue by miurahr** [miurahr/gdal#16]
There is a barrier to migrate CMake from GNUMake, Swig Perl script depends deeply. 

gdal/swig/perl/Makefile.PL includes followings:
```
my $source_tree = '../..';
unless (-r "$source_tree/GDALmake.opt.in") {
    die "../../GDALmake.opt.in was not found. Is this the GDAL source tree?";
}
```

**Comment by miurahr** [comment]
related: #7

**Comment by miurahr** [comment]
fixed by 1840174ef25a50b42813c22285840bfe45bf4c0e
